app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,59 @@
+*{
+  padding:0;
```

Comment:
Properties should be ordered margin, padding<br>Colon after property should be followed by one space